### PR TITLE
Add imoji to align with_friend text with top text

### PR DESCRIPTION
**Pull request by stomx**
image

### CODE REVIEW

**Comment by stomx** [comment]
Please check this request

File: Supporting FIles/ko.lproj/Localizable.strings [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 "wellcome_user_message" = "👋 안녕,  @${username} ${withFriend}";
 "with_friend" = "\n @${username}와 함께";
```
Comment (techinpark) [inline Supporting FIles/ko.lproj/Localizable.strings:10]: Do you mind if I don't erase that line?

**Review by techinpark — CHANGES_REQUESTED** [review]

File: Supporting FIles/ko.lproj/Localizable.strings (hunk 1 above)
Comment (techinpark) [reply Supporting FIles/ko.lproj/Localizable.strings:10]: I think `with_friend` key is duplicated

Comment (techinpark) [reply Supporting FIles/ko.lproj/Localizable.strings:10]: @stomx

**Pull request closed without merging**